src/main/features/core/tray.js

```diff
@@ -180,5 +180,50 @@ Emitter.on('audiooutput:list', (event, devices) => {
   setContextMenu();
 });
 
+// Change the Icon if its playing
+Emitter.on('playback:isPlaying', () => {
```

Comment:
sooo... not sure what would be better:
to have only the playing-icon show up when music is playing and the normal one if not
or
to have the playing-icon show up when music is playing and the paused icon if the music is paused.
because it seems like the playback api doesnt really wants to tell me if the music has stopped, thus the normal icon is really only visible until the mainpage has loaded and the playback api has fired that its not playing.
but then again, is there actually any way to flush your playbacklist so that the play/pause button is grey (as if you started GPMDP and havent listened to music)